db/migrate/20170420160820_copy_description_values_to_description_string.rb

```diff
@@ -0,0 +1,12 @@
+class CopyDescriptionValuesToDescriptionString < ActiveRecord::Migration[5.0]
+  def up
+    Edition.all.each do |edition|
+      string_description = edition.description.to_s
+      edition.update_attribute(:description_string, string_description)
+    end
+  end
+
+  def down
+    Edition.update_all(description_string: nil)
```

Comment:
I don't think we'll actually need a down method here, particularly as it would affect more than just this migration (all the values set from https://github.com/alphagov/publishing-api/pull/886)